ocs_ci/utility/csr.py

```
@@ -134,8 +135,113 @@ def get_nodes_csr():
+            logger.info(f"Rebooting {node_name} after successful rebase")
+            node.reboot()
+            fixed = True
+        except Exception as e:
+            logger.warning(
+                f"Failed to fix ostree image on {node_name} ({node_ip}): {e}"
+            )
+
+    return fixed
+
+
 def wait_for_all_nodes_csr_and_approve(
```

Ruff (0.16.1)

[warning] Too many branches (17 > 12)

(PLR0912)